src/main/java/formflow/library/FileController.java

```diff
@@ -36,17 +25,20 @@
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.*;
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.reactive.function.client.WebClientResponseException;
 import org.springframework.web.server.ResponseStatusException;
 import org.springframework.web.servlet.mvc.method.annotation.StreamingResponseBody;
 import org.springframework.web.servlet.view.RedirectView;
 
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.*;
```

Comment:
We generally try to avoid `.*` -- can you expand this out?   We've had to adjust a setting in intellij as well to stop it from automatically doing this.

Reply:
There is one on line 28 as well.

Reply:
I went through and converted all `.*` imports in the repo.

Reply:
I'll fix it. 

Wanted to point out that the pattern exists in the codebase - https://github.com/codeforamerica/form-flow/blob/74d4a62f4fe206cd6654fe393a7c3ecf9599eff2/src/test/java/formflow/library/inputs/TestFlow.java#L4 and https://github.com/codeforamerica/form-flow/blob/74d4a62f4fe206cd6654fe393a7c3ecf9599eff2/src/test/java/formflow/library/framework/ConditionalNavigationTest.java#L10. The IntelliJ setting might not be working perfectly. I'll fix it in these places as well.

Reply:
@coltborg beat me to it !!

Reply:
Ideally we wouldn't do that.  Those snuck past.   :)   Sometimes I don't complain about test code as it won't be part of the final jar. But I can start. ;)